### PR TITLE
Wip byo

### PR DESCRIPTION
**Pull request by aclevername**

### CODE REVIEW

File: docs/main/05-reference/06-statestore/01-statestore.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+# StateStores
+
+The Kratix `StateStore` resources represent the different backing storage options
+for Kratix to write to. Kratix supports both [GitStateStore](./02-gitstatestore.md) for writing to Git repositories,
+and [BucketStateStores](./03-bucketstatestore.md) for writing to S3-compatible buckets. When registering a
+[Cluster](../02-clusters/01-clusters.md) you specify which `StateStore` it should
+write to. Checkout the docs for the supported StateStores:
+
+- [GitStateStore](./02-gitstatestore.md)
```
Comment (kirederik) [inline docs/main/05-reference/06-statestore/01-statestore.md:15]: you could link as `./gitstatestore` instead of `02-gitstatestore.md`; that allows for renaming the file without having to rewrite everywhere the link is used.

NOTE: it does not work with partials :'(

File: docs/main/05-reference/02-clusters/01-clusters.md [hunk 2]
```diff
@@ -18,15 +20,60 @@ metadata:
...
+    kind: BucketStateStore
 ```
 
 When a new Cluster is registered in the Platform cluster (i.e., a new Cluster Resource is
-created), Kratix will create a pair of buckets in the [Repository](../01-deployment-topology/01-deployment-topology.md#gitops-repository): one for `resources`, one for `crds`. The
-full path of the buckets will be:
+created), Kratix will write to two paths in the [StateStore](../06-statestore/01-statestore.md):
+one for `resources`, one for `crds`. The path within the `StateStore` follows the following pattern:
+```
+<statestore.Spec.Path>/<cluster.Spec.Path>/<cluster.Metadata.Namespace>/<cluster.Metadata.Name>/
```
Comment (kirederik) [inline docs/main/05-reference/02-clusters/01-clusters.md:39]: this is a bit hard to parse...

suggestion:
```
statestore.Spec.Path/
    cluster.Spec.Path/
        cluster.Metadata.Namespace/
            cluster.Metadata.Name/
```

Comment (aclevername) [reply docs/main/05-reference/02-clusters/01-clusters.md:39]: yeah I struggled to write this out cleanly. I think thats an improvement, but its still not great!

File: docs/main/05-reference/06-statestore/01-statestore.md (hunk 1 above)
Comment (aclevername) [reply docs/main/05-reference/06-statestore/01-statestore.md:15]: @kirederik how does that work 🤔 is it using the ID from the page? does that mean even if the page moves to a different directory it works?
